src/backend/app.js

```diff
@@ -33,7 +33,7 @@ app.get('/future-meals', async (req, res) => {
     res.json(rows);
   } catch (e) {
     console.error(e);
-    res.status(400).json({ error: 'Internal server error' });
+    res.status(400).json({ error: 'Internal server error ' });
```

Comment:
You also can create variables for the error message, for example:
const errMessage = 'Internal server error'
and use it like:
res.status(400).json({ error: errMessage });